org.eclipse.m2e.core/src/org/eclipse/m2e/core/internal/project/registry/MavenProjectFacade.java

```diff
@@ -638,6 +664,10 @@ public IComponentLookup getComponentLookup() {
     return manager.getContainerManager().getComponentLookup(getConfiguration().getMultiModuleProjectDirectory());
   }
 
+  private void logException(CoreException e) {
```

Comment:
this does not log the exception it adds an error marker

Reply:
Same as above. Called by `IMavenProjectFacade.createExecutionContext()`.

Reply:
That's right. Changed it to addErrorMarker().